lib/fog/aws/models/compute/vpc.rb

```diff
@@ -106,7 +106,13 @@ def disable_classic_link_dns
 
         def save
           requires :cidr_block
-          data = service.create_vpc(cidr_block, { 'AmazonProvidedIpv6CidrBlock' => amazon_provided_ipv_6_cidr_block }).body['vpcSet'].first
+
+          options = {
+            'AmazonProvidedIpv6CidrBlock' => amazon_provided_ipv_6_cidr_block,
+            'InstanceTenancy' => tenancy
```

Comment:
There is an alias for the `instanceTenancy` above, do we need to alias `InstanceTenancy` as well?

Reply:
saw that alias and didn't really understand the use of it, could you please show me what you're thinking?

Basically what I'm trying to do here is pass the argument to create_vpc, we don't need anything beyond that.

Reply:
The value you are getting here from `tenancy` originates from that alias. Basically if you create a new model, when you pass in attributes, `InstanceTenancy` will get translated into the `tenancy` value due to that alias. But if you created a new vpc with `InstanceTenancy`, I'm not certain it would still set tenancy. Does that clarify?

Reply:
understood, and it certainly wouldn't
```
[17] pry(main)> Fog::AWS::Compute::VPC.new('instanceTenancy' => 'dedicated')
=>   <Fog::AWS::Compute::VPC
    id=nil,
    state=nil,
    cidr_block=nil,
    dhcp_options_id="default",
    tags=nil,
    tenancy="dedicated",
    is_default=nil,
    cidr_block_association_set=nil,
    ipv6_cidr_block_association_set=nil,
    amazon_provided_ipv6_cidr_block=false
  >
[18] pry(main)> Fog::AWS::Compute::VPC.new('InstanceTenancy' => 'dedicated')
=>   <Fog::AWS::Compute::VPC
    id=nil,
    state=nil,
    cidr_block=nil,
    dhcp_options_id="default",
    tags=nil,
    tenancy="default",
    is_default=nil,
    cidr_block_association_set=nil,
    ipv6_cidr_block_association_set=nil,
    amazon_provided_ipv6_cidr_block=false
  >
[19] pry(main)> 
```
However, it doesn't matter as the **response** is lower case, accordingly to the documentation at least https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Vpc.html
Right now my account is returning `Fog::AWS::Compute::Error: VpcLimitExceeded => The maximum number of VPCs has been reached.` so I can't really test this on Real.
I'll test as soon as I can clean some VPC which will take time, or maybe if you can test it yourself...

Reply:
If the response is lowercase it should probably be safe. If you are able to verify that would be great, I haven't actually directly run anything on AWS in ages (not even sure I still have valid creds at hand).

Reply:
FWIW, I think many of the AWS limits are soft-ish. ie if you have a reasonable use case/reason and ask nicely they can often bump limits for you.